src/main/resources/META-INF/rewrite/junit5.yml

```
@@ -51,6 +51,9 @@ description: Migrates JUnit 4.x tests to JUnit Jupiter.
 tags:
   - testing
   - junit
+preconditions:
+  - org.openrewrite.java.search.DoesNotUseType:
+      fullyQualifiedTypeName: org.testng..*
```

Did you consider adding `includeImplicit` ? Any pros/cons you see?
```suggestion
      fullyQualifiedTypeName: org.testng..*
      includeImplicit: true
```

---

I thought it might not be necessary, but after running a findtype with both options it seems to almost always result in more finding!